server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedIT.java

```diff
@@ -729,6 +732,10 @@ public void testSimpleNestedSortingWithNestedFilterMissing() throws Exception {
     }
 
     public void testNestedSortWithMultiLevelFiltering() throws Exception {
+        assumeFalse(
```

Comment:
are these tests still flaky?

Reply:
Unfortunately yes. It looks like there's some issue related to nested sorts, will track the fix in https://github.com/opensearch-project/OpenSearch/issues/11065